test/unit/attribute/hash_test.rb

```diff
@@ -77,33 +77,19 @@ def to_hash
         should "keep json strings" do
           assert_equal "{}", @ad.store("{}")
         end
-      end
-
-      context "store_hash_as_json as false" do
-        setup do
-          @attribute_handler = Aggregate::AttributeHandler.factory("testme", :hash, store_hash_as_json: false)
-        end
 
-        should "encode hash values as hashes" do
-          assert_equal({ a: 1 }, @attribute_handler.store(a: 1))
+        should "store nil as default empty JSON string" do
+          assert_equal "{}", @ad.store(nil)
         end
 
-        should "expand json strings as hashes" do
-          assert_equal({ "a" => 1 }, @attribute_handler.store("{\"a\":1}"))
-        end
-
-        should "set default value as empty hash" do
-          assert_equal({}, @attribute_handler.store(nil))
-        end
-
-        should "set empty string as default value" do
-          assert_equal({}, @attribute_handler.store(""))
+        should "store empty string as empty string" do
+          assert_equal "", @ad.store("")
         end
       end
 
-      context "aggregate_db_storage_type option as :elasticsearch" do
+      context "store_hash_as_json as false" do
```

Comment:
looks like we're missing a test case for where `store_hash_as_json` is explicit `true`